cocos/spine/skeleton-cache.ts

```diff
@@ -362,13 +362,12 @@ export class AnimationCache {
         _preFinalColor = null;
         _preDarkColor = null;
 
-        this.frames[index] = this.frames[index] || {
+        this.frames[index] ??= {
             segments: [],
             colors: [],
             boneInfos: [],
-            vertices: null,
-            uintVert: null,
-            indices: null,
+            vertices: new Float32Array(),
+            indices: new Uint16Array(),
```

Comment:
here is a known issue for TS that the type checking not work with `strictNullChecks` option

https://github.com/microsoft/TypeScript/issues/53162